map/framework.hpp

```diff
@@ -729,13 +729,21 @@ class Framework : public search::ViewportSearchCallback::Delegate,
   /// false otherwise.
   bool HasRouteAltitude() const;
   /// \brief Generates 4 bytes per point image (RGBA) and put the data to |imageRGBAData|.
+  /// \param width is width of chart shall be generated in pixels.
+  /// \param height is height of chart shall be generated in pixels.
+  /// \param imageRGBAData is bits of result image in RGBA.
+  /// \param minRouteAltitude is min altitude along the route in altitudeUnits.
```

Comment:
optional. let add that is output parameter. it may be obvious from the code, but would be better to add to comment, i think.